system/doc/reference_manual/errors.md

```diff
@@ -30,14 +30,14 @@ Errors can roughly be divided into four different types:
   crash. An example is that nothing happens when a button in a graphical user
   interface is clicked.
 
-- **[](){: #run-time-errors } Run-time errors**  
+- **[](){: #run-time-errors } Run-time errors**
```

Comment:
The logic I used for translating definition lists to markdown is that either it is translated to:

```
- **tag** - value
```

or 

```
- **tag**  <!-- note the double space >
  value
```

So to stay consistent I would add:

```suggestion
- **[](){: #run-time-errors } Run-time errors** - 
```

You can also cleanup the anchor if you want to this:

```suggestion
- **Run-time errors**{: #run-time-errors } - 
```